packages/eslint-config-eventbrite-legacy/rules/style.js

```diff
@@ -22,6 +26,11 @@ module.exports = {
         // http://eslint.org/docs/rules/consistent-this
         'consistent-this': ['error', 'self'],
 
+        // require function names to match the name of the variable or
+        // property to which they are assigned
+        // https://eslint.org/docs/rules/func-name-matching
```

Comment:
man so many rules i've never even given a second thought about hah

Reply:
😂 I agree, some of these I just expect to be the case.

Reply:
i know right! just obvious things, but apparently not 100% obvious. _someone_ created the rule to combat some "bad" behavior